manual/command-reference-manual.tex

```diff
@@ -4244,9 +4244,15 @@ \section{synth\_xilinx -- synthesis for Xilinx FPGAs}
         is omitted if this parameter is not specified.
 
     -vpr
-        generate an output netlist (and BLIF file) suitable for VPR
+        generate an output netlist (and BLIF file) suitable for VPR.
         (this feature is experimental and incomplete)
 
+    -no-brams
```

Comment:
(nitpick) Tradition in Yosys is to use -nobram, -nodram, etc rather than -no-brams

Reply:
Fixed